addons/mail/static/src/models/chat_window_manager/chat_window_manager.js

```diff
@@ -137,7 +137,6 @@ function factory(dependencies) {
             _newOrdered[index] = otherChatWindow.localId;
             _newOrdered[index + 1] = chatWindow.localId;
             this.update({ _ordered: _newOrdered });
-            chatWindow.focus();
```

Comment:
Why remove this? It was good to keep focus on chat window when clicking on shift.

Reply:
That's a fail, I wanted to remove the if I added